src/SonarScanner.MSBuild.PreProcessor/WebServer/SonarWebServer.cs

```diff
@@ -177,33 +177,33 @@ protected virtual void Dispose(bool disposing)
             return await DownloadComponentProperties(component);
         }
 
-        protected async Task<T> ExecuteWithLogs<T>(Func<Task<T>> request, Uri logUri)
+        protected async Task<T> ExecuteWithLogs<T>(Func<Task<T>> request)
         {
             try
             {
                 return await request();
             }
             catch (Exception e)
             {
-                logger.LogError("Failed to request and parse '{0}': {1}", logUri, e.Message);
+                logger.LogError("Failed to request and parse: {0}", e.Message);
```

Comment:
This does not help us much. The url will never be populated.

How are the logs now in case of an error. Do they help us identify the issue?